Tests/TestingTests/Test.Case.Argument.IDTests.swift

```diff
@@ -47,10 +49,11 @@ struct Test_Case_Argument_IDTests {
 
   @Test("One Identifiable parameter")
   func oneIdentifiableParameter() async throws {
-    let test = Test(
+    var test = Test(
       arguments: [MyIdentifiableArgument(id: "abc")],
       parameters: [Test.Parameter(index: 0, firstName: "value", type: MyIdentifiableArgument.self)]
     ) { _ in }
+    try await test.evaluateTestCases()
```

Comment:
Should we just have the fixture `Test.init` call this? It would make the overall change more concise.

Reply:
I have re-organized things a bit so that we no longer need to call this on individual fixture test instances